.github/actions/contentful-checks/action.yml

```diff
@@ -42,14 +42,23 @@ runs:
         CONTENTFUL_SPACE_ID: ${{ inputs.contentful-space-id }}
         CONTENTFUL_ENV_ID: ${{ inputs.contentful-master-env }}
         CONTENTFUL_MANAGEMENT_ACCESS_TOKEN: ${{ inputs.contentful-token }}
+      # Determine whether the Contentful webhook setup has changed since master
+    - uses: technote-space/get-diff-action@v6
+      with:
+        PATTERNS: |
+          +packages/contentful/scripts/setup-webhook.ts
```

Comment:
Is there a case for adding `packages/contentful/migrations` to this path and getting rid of the migration count logic? That would help the problem with false positives when CI fails on master.

Happy to consider that question OOS for this PR though.

Reply:
interesting.. at first I thought that what we were only interested in the migrations that have not been run yet, and not a diff, especially when fails in the pipeline or if the migration itself is broken and fails. But then I thought - so what, it's not like we want a dedicated environment in that case anyway. 

The only other issue I can think of is that the diff does not say if something has been added, it just tells you if its different from master, and again I don't know if that matters. Obvs its OOS for this PR but 100% worth the discussion, ill create a bug-ticket to address the issue with unnecessary environment creation and we can tackle it there, does this work for you @lennym ?